metainfo.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/dists/org.scummvm.scummvm.metainfo.xml b/dists/org.scummvm.scummvm.metainfo.xml
+index c8da017405b..dab6aef09cd 100644
+--- a/dists/org.scummvm.scummvm.metainfo.xml
++++ b/dists/org.scummvm.scummvm.metainfo.xml
+@@ -1,7 +1,7 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <!-- Copyright 2020-2023 The ScummVM Team -->
+ <component type="desktop-application">
+-  <id>org.scummvm.scummvm</id>
++  <id>org.scummvm.ScummVM</id>
+   <metadata_license>CC0-1.0</metadata_license>
+   <project_license>GPL-3.0-or-later</project_license>
+   <name>ScummVM</name>
+@@ -26,7 +26,7 @@
+   <summary xml:lang="ru">Интерпретатор множества приключенческих и ролевых игр</summary>
+   <summary xml:lang="uk">Інтерпретатор для численних пригодницьких і рольових ігор</summary>
+   <developer_name>The ScummVM Team</developer_name>
+-  <launchable type="desktop-id">org.scummvm.scummvm.desktop</launchable>
++  <launchable type="desktop-id">org.scummvm.ScummVM.desktop</launchable>
+   <provides>
+     <binary>scummvm</binary>
+   </provides>
+@@ -332,4 +332,18 @@ Blade Runner та незліченної кількості інших, ви з
+     <content_attribute id="money-purchasing">none</content_attribute>
+     <content_attribute id="money-gambling">none</content_attribute>
+   </content_rating>
++  <releases>
++    <release version="2.8.0" date="2023-12-16"/>
```

Comment:
The date will probably require adjustment to match the actual release date.

Reply:
Yes, this should be changed. The planned release date for 2.8.0 is 2023-12-30.

Reply:
Thanks!